### PR TITLE
CFD-385: Group management Categories : Add/Update/Delete category. Reorder categories.

### PR DESCRIPTION
**Pull request by solker**
#592
#599

### CODE REVIEW

**Comment by HelenaEksler** [comment]
@nader77 Review this, please.

**Comment by HelenaEksler** [comment]
video13

File: capacity4more/modules/c4m/features_og/c4m_features_og_manage/c4m_manage_categories.inc [hunk 1]
```diff
@@ -55,7 +55,14 @@ function c4m_taxonomy_overview_terms($form, &$form_state, $vocabulary) {
 
   $delta = 0;
   $term_deltas = array();
-  $tree = taxonomy_get_tree($vocabulary->vid);
+  $menu_item = menu_get_item(current_path());
+  if ($menu_item['path'] == 'manage/categories/types') {
```
Comment (nader77) [inline capacity4more/modules/c4m/features_og/c4m_features_og_manage/c4m_manage_categories.inc]: This can be done in a one line condition.

**Comment by nader77** [comment]
@HelenaEksler Everything seems to be working ok, Just please check for unneeded code from the copying in ``c4m_manage_categories.inc`` that can be deleted to optimize code.

**Comment by HelenaEksler** [comment]
@nader77 Ready.

**Comment by HelenaEksler** [comment]
@nader77 We can merge it into the parent branch and assign that PR to Marteen?

**Comment by nader77** [comment]
Yea merging.

**Pull request merged**